bunsen/bunsen-core/src/main/java/com/cerner/bunsen/definitions/PrimitiveConverter.java

```diff
@@ -63,7 +63,13 @@ protected Object fromHapi(IPrimitiveType primitive) {
 
   @Override
   public Object fromHapi(Object input) {
-
+    // TODO: remove this hack! It is added to address this bug:
+    // https://github.com/google/fhir-data-pipes/issues/1014
+    // with root cause being:
+    // https://github.com/hapifhir/org.hl7.fhir.core/issues/1800
+    if (!(input instanceof IPrimitiveType)) {
+      return "";
```

Comment:
With this change I am assuming that we add a empty string for the `div` element, where as earlier we were not returning the element itself.

Reply:
Correct, but please also note that in the Search API mode, currently we set `_summary=data` (e.g., [here](https://github.com/google/fhir-data-pipes/blob/f0d0a6c955a69f71a9ac3bfed2a4e0a72ec2e8b1/pipelines/batch/src/main/java/com/google/fhir/analytics/FetchResources.java#L144)) when we are fetching resources; so we have no `text` field.